distribution/docs/src/main/resources/_contents/_configuring/configuring-landing-page-contents.adoc

```diff
@@ -1,14 +1,14 @@
 
-=== Landing Page
+==== Configuring the Landing Page
 
 The ${branding} landing page offers a starting point and general information for a ${branding} node.
 It is accessible at `/(index|home|landing(.htm|html))`.
 
-==== Installing the Landing Page
+===== Installing the Landing Page
 
-The Landing Page is is not installed by default with a standard installation.
+The Landing Page is installed by default with a standard installation.
 
-==== Configuring the Landing Page
+===== Editing the Landing Page
```

Comment:
I do like "editing", but how do you feel about "customizing", here? 
Not going to make or break it for me either way.  :)